.github/workflows/extension-release.yml

```
@@ -0,0 +1,240 @@
+            - Install from VS Code Marketplace: [Task Master Kanban](https://marketplace.visualstudio.com/items?itemName=[TBD])
+            - Install from Open VSX Registry: [Task Master Kanban](https://open-vsx.org/extension/[TBD])
+            - Or download the VSIX file below and install manually
+          draft: false
+          prerelease: false
+
+      - name: Upload VSIX to Release
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
```

actionlint (1.7.7)

property "create_release" is not defined in object type {vsix-info: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)